src/ninja.cc

```diff
@@ -1087,6 +1096,124 @@ int NinjaMain::ToolRestat(const Options* options, int argc, char* argv[]) {
   return EXIT_SUCCESS;
 }
 
+struct Compdb_Targets {
```

Comment:
a few nit related to stylistic issues:

- Do not use underscores in type names.

- Use ALL_CAPS for constants in enum values, or as an alternative `kCamelCase` (the `k` prefix is an indication that this is a constant, not not a method).

- Use CamelCase for method and functions, except for simple getters and setters, and when trying to mimic standard library classes. E.g. `StringPiece::size()`.